### ISSUE TITLE
Can't build from master

### ISSUE THREAD

**Issue by mickaelistria** [OmniSharp/omnisharp-node-client#436]
I can't build project from master, from a clean repo

```
$ npm install

...
> omnisharp-client@7.0.6 prepublish /home/mistria/git/omnisharp-node-client
> tsc
lib/drivers/StdioDriver.ts(219,19): error TS7006: Parameter 'ctx' implicitly has an 'any' type.
lib/drivers/StdioDriver.ts(225,19): error TS7006: Parameter 'runtime' implicitly has an 'any' type.
lib/helpers/runtime.ts(247,43): error TS7024: Function implicitly has return type 'any' because it does not have a return type annotation and is referenced directly or indirectly in one of its return expressions.
test/omnisharp-runtime-spec.ts(71,22): error TS7031: Binding element 'runtime' implicitly has an 'any' type.
test/omnisharp-runtime-spec.ts(71,31): error TS7031: Binding element 'path' implicitly has an 'any' type.
test/omnisharp-runtime-spec.ts(84,22): error TS7031: Binding element 'runtime' implicitly has an 'any' type.
test/omnisharp-runtime-spec.ts(84,31): error TS7031: Binding element 'path' implicitly has an 'any' type.
...
```

Is this an issue on my side?
cc @AndrienkoAleksandr @david-driscoll

**Comment by AndrienkoAleksandr** [comment]
I had the same problem and  fixed this in my fork https://github.com/AndrienkoAleksandr/omnisharp-node-client/commit/ed475537b087a2776ba22c06e20f0e3b8902e928  But actually I think it's can depends on node version or tslint settings...

**Comment by mickaelistria** [comment]
I've updated the version my typescript version to 2.4.1, and get the same issue, I'll try your patch.
What would you think about having us trying to maintain a joined fork of this project? The patch I need for it to work in my case is #377

**Comment by AndrienkoAleksandr** [comment]
I think we can try idea with fork.

**Comment by AndrienkoAleksandr** [comment]
But by the way if you need latest version omnisharp-client without compilation you can do it with help npm:
create some folder, go to this folder with help terminal, type command "npm init", When you will get proposal for project name type something, for another proposals type "Enter" and in the end "Yes". You will get package.json in the folder and folder initialized like npm project. After that type "npm install --save omnisharp-client@7.0.6" and after that you will get node modules folder with compiled node_modules. This node_modules npm gets from release https://www.npmjs.com/package/omnisharp-client . This node_modules is working omnisharp-client :)

**Comment by david-driscoll** [comment]
I'll fix this up for 2.4.1 on the weekend, sorry for the problems!

**Comment by mickaelistria** [comment]
Works for me now, thanks.
